### PR TITLE
Full Test Data

### PR DESCRIPTION
**Pull request by nickhsmith**
update the test_full samplesheet to be full open source WXS data from GIAB and NCBI

### CODE REVIEW

File: conf/test_full.config [hunk 1]
```diff
@@ -17,11 +17,7 @@ params {
     // Input data for full size test
     // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
     // TODO nf-core: Give any required params for the test so that command line flags are not needed
```
Comment (maxulysse) [inline conf/test_full.config:19]:
```suggestion
```

File: conf/test_full.config [hunk 2]
```diff
@@ -17,11 +17,7 @@ params {
...
-    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_full_illumina_amplicon.csv'
-
-    // Input data for full size test
-    // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
+    input = "${baseDir}/tests/csv/3.0/test_full_data.csv"
 
     // Ignore `--input` as otherwise the parameter validation will throw an error
     schema_ignore_params = 'genomes,input,modules'
```
Comment (maxulysse) [inline conf/test_full.config:23]:
```suggestion
    schema_ignore_params = 'genomes'
```

**Comment by nickhsmith** [comment]
I will say that these are real life examplse of WXS, but we could down-sample or find sequencing runs with less coverage if the storage/compute time is too large.

**Comment by maxulysse** [comment]
If you can just update the CHANGELOG that would be perfect.
I'd say we need to run it once to figure out if we need to down-sample

**Comment by FriederikeHanssen** [comment]
@nickhsmith from what publication is the paired data coming from?

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [dev](https://github.com/nf-core/sarek/tree/dev)
 
+### Changed
+- [#580] (https://github.com/nf-core/sarek/pull/580) - changed the test_full config to real public WXS data. 1 sample WXS germline, 1 Tumor/Normal pair. 
+
```
Comment (maxulysse) [inline CHANGELOG.md:12]:
```suggestion
```

There's already a Changed subsection for `dev`

**Comment by nickhsmith** [comment]
> @nickhsmith from what publication is the paired data coming from?

Good point. I'll add that to the log

File: CHANGELOG.md [hunk 4]
```diff
@@ -33,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- [#580] (https://github.com/nf-core/sarek/pull/580) - changed the test_full config to real public WXS data. 1 sample WXS germline, 1 Tumor/Normal pair. https://doi.org/10.1038/sdata.2016.25 and https://doi.org/10.1038/s41587-021-00994-5
```
Comment (maxulysse) [inline CHANGELOG.md:33]:
```suggestion
- [#580](https://github.com/nf-core/sarek/pull/580) - changed the test_full config to real public WXS data. 1 sample WXS germline, 1 Tumor/Normal pair. https://doi.org/10.1038/sdata.2016.25 and https://doi.org/10.1038/s41587-021-00994-5
```

**Comment by FriederikeHanssen** [comment]
Wondering if this large file staging will work. Might have to upload htem to the bucket directly and then change the paths

**Comment by nickhsmith** [comment]
> Wondering if this large file staging will work. Might have to upload them to the bucket directly and then change the paths

We can move them to whichever bucket we'd like
https://www.ncbi.nlm.nih.gov/Traces/cloud-delivery/

**Comment by maxulysse** [comment]
@nf-core-bot fix linting

**Pull request merged**